ONSdigital/eq-questionnaire-runner
# Add optional auth token for address lookups

### What is the context of this PR?
Adds an optional auth token to address lookups. This is only added to the page context if an `Address` answer with lookup options exists.

The duration of the token uses the default session timeout duration, rather than a schema defined one. This is simpler as there is no need to access the schema when we're generating the token, and there is no requirement for the address token to expire any sooner than the default session timeout. Also, a custom timeout is only used in the test timeout schema.

### How to review 
- Check out and change the `ADDRESS_LOOKUP_API_AUTH_ENABLED` setting to `True`
- Run the `test_address_lookups` schema
- Check that the `address-mandatory-autosuggest-container` attribute in the HTML source is set to a JWT

### Checklist
